doc/user/content/integrations/sql-clients.md

```diff
@@ -17,7 +17,136 @@ with many SQL clients that support PostgreSQL (see [Tools and Integrations](/int
 In this guide, we’ll cover how to connect to your Materialize region using
 common SQL clients.
 
-## `psql`
+## Connection parameters
+
+You can find the credentials for your Materialize region in the
+[Materialize console](https://console.materialize.com/), under **Connect
+externally** in the navigation bar.
+
+Field             | Value
+----------------- | ----------------
+Host              | Materialize host name.
+Port              | **6875**
+Database name     | Database to connect to (default: **materialize**).
+Database username | Materialize user.
+Database password | App password for the Materialize user.
+SSL mode          | **Require**
+
+Before connecting, double-check that you've created an **app-password** for your
+user. This password is auto-generated, and prefixed with `mzp_`.
+
+### Runtime connection parameters
+
+{{< warning >}}
+Parameters set in the connection string work for the **lifetime of the
+session**, but do not affect other sessions. To permanently change the default
+value of a session variable for a specific user (i.e. role), use the
+[`ALTER ROLE...SET`](/sql/alter-role) command.
+{{< /warning >}}
+
+You can pass runtime connection parameters (like `cluster`, `isolation_level`,
```

Comment:
I just want to note that users might not find these session variable options if they are looking in the sql drivers section to connect programmatically. Maybe we could link here in the sql driver pages so they can see how to include the more advanced options in the connection string

Reply:
Will do that as a follow-up!